fingertip/util/filesystem.py

```diff
@@ -0,0 +1,99 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
+# Copyright (c) 2020 Red Hat, Inc., see CONTRIBUTORS.
+
+"""
+Helper functions for fingertip: Setup and cleanup filesystem
+"""
+
+import os
+import sys
+import shutil
+import subprocess
+
+from fingertip.util import log
+from fingertip.util.path import MACHINES, CACHE
+from fingertip.util.reflink import is_supported
+
+
+SETUP = os.getenv('FINGERTIP_SETUP', 'suggest')
+SIZE = os.getenv('FINGERTIP_SETUP_SIZE', '25G')
+
+
+def create_supported_fs(backing_file, size):
+    subprocess.run(['fallocate', '-l', size, backing_file], check=True)
+    subprocess.run(['mkfs.xfs', '-m', 'reflink=1', backing_file],
+                   check=True)
+
+
+def mount_supported_fs(backing_file, tgt):
+    # use warning to avoid rewriting the prompt with sudo prompt
```

Comment:
huh? If it's to workaround my fancy logging handler, it's better to disable it with `fingertip.log.plain()`.

Reply:
Somewhere around the entry point (currently filesystems.main).